src/components/Todo/Todo.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { User } from '../User';
+import './Todo.scss';
+
+export const Todo = ({ title, completed, user }) => (
+  <div className="todoCard">
+    <h2>{title}</h2>
+    <p>{completed}</p>
```

Comment:
```suggestion
        <p>{`Completed: ${completed}`}</p>
```
React не рендерит boolean.

Reply:
Thanks, I`ll fix it